app/javascript/src/components/Player.tsx

```diff
@@ -37,6 +49,8 @@ class Player extends React.Component<PlayerProps, PlayerState> {
 
     this.engine.installDefaultFactories()
     this.model.deSerializeDiagram(this.props.story, this.engine)
+
+    import(`./themes/${props.theme}Theme.css`)
```

Comment:
could u use this to delay rendering or something like that on this (not sure how the thing all works so maybe not necessary) just always see this kind of import then utilized as a promise

Reply:
Ya. We definitely could. I should take a second pass at this. These could just as easily be included in the head of the document.